crates/environ/src/compile/stack_maps.rs

```diff
@@ -0,0 +1,175 @@
+use crate::obj::ELF_WASMTIME_STACK_MAP;
+use crate::prelude::*;
+use cranelift_bitset::CompoundBitSet;
+use object::write::{Object, StandardSegment};
+use object::{LittleEndian, SectionKind, U32Bytes};
+
+/// Builder for the `ELF_WASMTIME_STACK_MAP` section in compiled executables.
+///
+/// This format is parsed by `crate::stack_map`.
+///
+/// The current layout of the format is:
+///
+/// * A 4-byte little-endian count of how many stack maps there are: `N`.
+/// * `N` 4-byte little endian program counters, in ascending order.
+/// * `N` 4-byte little endian offsets.
+/// * Stack map data as 4-bit little endian integers.
```

Comment:
```suggestion
/// * Stack map data as 4-byte little endian integers.
```
Without looking below to see what this should be... is this correct?

Reply:
Whoops, yes!